modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFListing.macro

```diff
@@ -2,10 +2,14 @@ definition {
 
 	@summary = "Verify if the MDF is visible on listing page"
 	macro viewOnList {
-		AssertVisible(
-			key_rowNumber = 6,
-			key_rowValue = ${campaignName},
-			locator1 = "PRMMDFListing#LISTING_TABLE");
+		var key_rowName = ${key_rowNumber};
+
+		if (${rowNumber} == "Status") {
+			AssertVisible(
+				key_rowNumber = "Status",
+				key_rowValue = ${mdfStatus},
+				locator1 = "PRMMDFListing#LISTING_TABLE");
+		}
```

Comment:
Hi @mateussandes23, I think that this macro is incomplete. I asked you to add a condition for each column. Do you remember?

Reply:
You should update the [commit message](https://github.com/pt-liferay-solutions/liferay-portal/pull/3228/commits/bed55d14d6c9d4b6bc32aae699b42173bd4f1a02#) too.